public/js/app/components/ActionModalCourseRounds.jsx

```diff
@@ -173,18 +173,21 @@ function ActionModalCourseRounds(props) {
           <Label htmlFor="choose-rounds-list">{info.chooseRound.addRounds.infoText}</Label>
           <Form className={`Available--Rounds--To--Add ${alert && alert.isOpen ? 'error-area' : ''}`}>
             {availableRounds.map(round => (
-              <FormGroup className="form-check" id="choose-rounds-list" key={'add' + round.ladokRoundId}>
+              <FormGroup className="form-check" id="choose-rounds-list" key={'add' + round.applicationCodes[0]}>
```

Comment:
What happens if no application code is found in applicationCodes? it should be first sure that application code exists under index 0

Reply:
Changed array to direct string. Now round.applicationCode will be used.